lib/scrum/priority_name.rb

```diff
@@ -0,0 +1,15 @@
+module Scrum
+  class PriorityName
+    PRIORITY_REGEX      = /^(?:\([\d.]+\) )?P(\d+): /
+
+    def priority(name)
+      return unless m = name.match(PRIORITY_REGEX)
+      m.captures.first.to_i
+    end
+
+    def build(name, n)
```

Comment:
This method implies that there should be no card which doesn't have priority set but has an estimation. To explain:

- Lets say I have this card "(3) Prepare dinner for my beloved one"
- I pass it through ```build``` method here:

```
  card.name = priority_name.build(name, 8)
```
- Now ```card.name``` is "P8: (3) Prepare dinner for my beloved one" which unfortunately doesn't match the priority regex (although ```build``` was supposed to return a "valid" name).

We can either create a convention that we don't support card with estimation but no priority (but I'm not sure if it fits all the flows) or change this method a bit so it takes care of that case too.

Whichever the case, I think a test should be added, that will check that either an error is returned when a card with an invalid name is detected or that the case is handled gracefully.